indico/modules/events/management/client/js/index.js

```diff
@@ -220,7 +220,11 @@ import {SeriesManagement} from './SeriesManagement';
 
     $('.js-event-person-list td').on('mouseenter', function() {
       const $this = $(this);
-      if (this.offsetWidth < this.scrollWidth && !$this.attr('title')) {
+      if (
+        this.offsetWidth < this.scrollWidth &&
+        !$this.attr('title') &&
+        !$this.find('.popup-container').length
```

Comment:
i'd use `js-popup-container` to make it clear it's not for styling but just for js.